clients/admin-ui/next.config.js

```diff
@@ -42,9 +41,13 @@ const nextConfig = {
       },
     ];
   },
-  images: {
-    loader: "custom",
-  },
 };
 
+// Export the static site for production.
+// Rewrites are not supported in static sites.
+if (process.env.PROD_EXPORT === "true") {
+  delete nextConfig.rewrites;
+  nextConfig.output = "export";
+}
```

Comment:
these changes are needed because `next export` has been removed in favor of `output: "export"`.